lib/huk_load/huk_load.c

```diff
@@ -0,0 +1,102 @@
+#include <sys/printk.h>
+#include <pm_config.h>
+#include <string.h>
+
+#include <hal/nrf_acl.h>
+#include <hal/nrf_ficr.h>
+
+#include <nrf_cc3xx_platform.h>
+#include <nrf_cc3xx_platform_mutex.h>
+#include <nrf_cc3xx_platform_abort.h>
+/* The nrf52840 doesn't have a KMU but the helper functions handling the KDR
+ *  key is inside the following header
+ */
+#include <nrf_cc3xx_platform_kmu.h>
+
+/* The magic data identify the flash page which contains the key */
+static uint8_t magic_data[16] = { 0x56, 0xB5, 0x6D, 0xAD,
+						0xAB, 0x5B, 0x6A, 0xB5,
+						0x55, 0xAB, 0x5A, 0xDA,
+						0xAB, 0x55, 0xAD, 0x5A };
+
+static uint32_t huk_addr = (uint32_t)PM_HUK_PARTITION_ADDRESS;
+
+
+/* Initialize the platform apis without RNG support */
+int init_cc3xx_platform()
+{
+	int res;
```

Comment:
nitpicking here, but I think we should try and use the most common name for our error variables. The most popular is ... drumroll ...

```
~/ncs/nrf tmp-fmfu*
❯ rg -t c "int err" | wc
   1359    5153   82303

~/ncs/nrf tmp-fmfu*
❯ rg -t c "int ret" | wc
    277    1121   17683

~/ncs/nrf tmp-fmfu*
❯ rg -t c "int res" | wc
     21     107    1841


```

Reply:
Hahahaha, sure I will change that to err then :D